webpack.config.js

```diff
@@ -9,7 +9,7 @@ const IS_DEV = process.env.NODE_ENV === 'development'
 const plugins = [
   new Webpack.DefinePlugin({
     'process.env': {
-      'NODE_ENV': IS_DEV ? JSON.stringify('development') : JSON.stringify('production')
+      NODE_ENV: JSON.stringify(process.env.NODE_ENV)
```

Comment:
Your way is better. I actually did it this way while working on webpack at work last week.

Reply:
:thumbsup